3 changes

dipranjan opened the pull request:
1. Loss function note added - ch7
2. Chain rule example added - appendix math
3. Sigmoid function image added - ch7

Might need to do a round of QC once, I am not extremely comfortable in Latex, still tried my best

dipranjan committed:
Merge pull request #1 from dafriedman97/gh-pages

merge

dipranjan committed:
Create concept.md

dipranjan committed:
Add files via upload

dipranjan committed:
Loss function Note added, sigmoid image added

dipranjan committed:
chain rule example added

dafriedman97 commented:
For some reason these changes are looking a little funny on the "files changed" view (I can't see the side by side comparisons--seems to be showing that ~200 lines are being added). I might add these changes manually if that's all right?

dipranjan commented:
Yeah actually the problem is that  your gh-pages branch does not have any concept.md file. So since I wanted to make a change in that file I had to copy the file from the master branch and then add the change to it. Which is why it is showing 200 lines added as essentially the entire file is new to the gh-pages branch.

If you plan on merging the changes via gh-pages branch please ensure that master and gh-page branches are in sync. You can merge 200 lines it should not be a problem as it has the same content as the master branch.

dipranjan committed:
Merge pull request #2 from dafriedman97/gh-pages

Update documentation

dipranjan committed:
Updated Adaboost

1. Added a common sense introduction of adaboost along with the original paper
2. Added an example of the adaboost calculation
3. Updated the formulae of the weight update
4. Added a note on Outliers and weight update

dipranjan commented:
You have already made the changes
